packages/php-wasm/universal/src/lib/base-php.ts

```diff
@@ -616,6 +616,53 @@ export abstract class BasePHP implements IsomorphicLocalPHP {
 		this[__private__dont__use].FS.rename(fromPath, toPath);
 	}
 
+	/** @inheritDoc */
+	@rethrowFileSystemError('Could not copy "{path}"')
+	cp(fromPath: string, toPath: string, recursive = false) {
+		const FS = this[__private__dont__use].FS;
+
+		const fromStat = FS.stat(fromPath);
+
+		// Emscripten directories will have the sticky-bit set
+		// https://linux.die.net/man/1/chmod
+		const fromIsDir = fromStat.mode & 0o40000;
+
+		let toExists: boolean;
+
+		try {
+			FS.stat(toPath);
+			toExists = true;
+		} catch {
+			toExists = false;
+		}
+
+		if (!fromIsDir) {
+			const file = FS.readFile(fromPath, { encoding: 'binary' });
+			FS.writeFile(toPath, file);
```

Comment:
What should happen if `toPath` exists and is a directory?

Reply:
@adam In that case, `it('cp() should copy a file into a directory', () => {...`

https://github.com/WordPress/wordpress-playground/blob/8da67082fc0347b6a1d7c94c6f1506be3316666c/packages/php-wasm/universal/src/lib/base-php.ts#L658-L665

https://github.com/WordPress/wordpress-playground/blob/8da67082fc0347b6a1d7c94c6f1506be3316666c/packages/php-wasm/node/src/test/php.spec.ts#L351-L360